src/main/java/de/thomas_oster/liblasercut/drivers/Ruida.java

```diff
@@ -92,6 +92,8 @@ public class Ruida extends LaserCutter
   protected static final String SETTING_BED_WIDTH = "Bed width (mm)";
   protected static final String SETTING_BED_HEIGHT = "Bed height (mm)";
   protected static final String SETTING_USE_BIDIRECTIONAL_RASTERING = "Use bidirectional rastering";
+  protected static final String SETTING_RASTER_PADDING = "Padding per raster line (mm)";
+  protected static final String SETTING_RASTER_OUTSIDE = "Allow padding outside of specs";
```

Comment:
should be the same texts as in https://github.com/t-oster/LibLaserCut/blob/5215383458060acb7404d01356a2a8af0137f52c/src/main/java/de/thomas_oster/liblasercut/drivers/GenericGcodeDriver.java#L109-L110 unless there is a very good reason. If the other text is not good, feel free to change both.